pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -279,6 +279,7 @@ type GameServerAllocationStatus struct {
 	Ports          []agonesv1.GameServerStatusPort `json:"ports,omitempty"`
 	Address        string                          `json:"address,omitempty"`
 	NodeName       string                          `json:"nodeName,omitempty"`
+	IsRemote       bool                            `json:"isRemote"`
```

Comment:
Drive-by bikeshed comment, but this is already a bool. Should it just be `Remote`? The `Is` seems redunandt.

Reply:
It seems we have mixed pattern within the code base, eg [here](https://github.com/googleforgames/agones/blob/main/cmd/sdk-server/main.go#L312) and [here](https://github.com/googleforgames/agones/blob/main/pkg/apis/allocation/v1/gameserverallocation.go#L234). I will remove `Is` to align with the other bool in the same file.